Workspaces/Base/WorkspaceUI.cs

```diff
@@ -245,10 +245,12 @@ public void OnDragging()
 						+ transform.forward * (absForward - (currentExtents.z - extents.z)) * Mathf.Sign(positionOffsetForward);
 
 					m_WorkspaceUI.transform.parent.position = m_PositionStart + positionOffset * viewerScale;
+					m_WorkspaceUI.Pulse(rayOrigin, 0.25f, 0.35f, true, true);
```

Comment:
Another case of haptics layering up over each frame of interaction - is this intentional?